packages/from-query/__tests__/__snapshots__/from-query-test.ts.snap

```diff
@@ -717,11 +717,6 @@ export interface IFragmentTypeRef {
    */
   INLINE_FRAGMENT = 'INLINE_FRAGMENT',
 
-  /**
```

Comment:
I am not quite sure why this was removed. I see it was added in here: https://github.com/avantcredit/gql2ts/pull/222